client/src/app/site/motions/modules/motion-poll/motion-poll-detail/motion-poll-detail.component.ts

```diff
@@ -0,0 +1,166 @@
+import { Location } from '@angular/common';
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { MatSnackBar } from '@angular/material';
+import { Title } from '@angular/platform-browser';
+import { ActivatedRoute, Router } from '@angular/router';
+
+import { TranslateService } from '@ngx-translate/core';
+import { Observable } from 'rxjs';
+
+import { MotionPollRepositoryService } from 'app/core/repositories/motions/motion-poll-repository.service';
+import { GroupRepositoryService } from 'app/core/repositories/users/group-repository.service';
+import { MotionPoll } from 'app/shared/models/motions/motion-poll';
+import { BaseViewComponent } from 'app/site/base/base-view';
+import { ViewMotionPoll } from 'app/site/motions/models/view-motion-poll';
+import { ViewGroup } from 'app/site/users/models/view-group';
+
+@Component({
+    selector: 'os-motion-poll-detail',
+    templateUrl: './motion-poll-detail.component.html',
+    styleUrls: ['./motion-poll-detail.component.scss']
+})
+export class MotionPollDetailComponent extends BaseViewComponent implements OnInit {
+    private pollId: number;
+
+    public pollTypes: object = { analog: 'Analog', named: 'Named', pseudonymous: 'Pseudonymous' };
+
+    public pollMethods: object = { YN: 'Yes/No', YNA: 'Yes/No/Abstain' };
+
+    public percentBase: object = {
+        YN: 'Yes/No per candidate',
+        YNA: 'Yes/No/Abstain per candidate',
+        valid: 'All valid ballots',
+        cast: 'All casted ballots',
+        disabled: 'Disabled (no percents)'
+    };
+
+    public majority: object = {
+        simple: 'Simple majority',
+        two_thirds: 'Two-thirds majority',
+        three_quarters: 'Three-quarters majority',
+        disabled: 'Disabled'
+    };
+
+    public stateType: object = {
+        1: 'Created',
+        2: 'Started',
+        3: 'Finished',
+        4: 'Published'
+    };
+
+    public userGroups: ViewGroup[] = [];
+
+    public groupObservable: Observable<ViewGroup[]> = null;
+
+    public isNewPoll = false;
+
+    public poll: ViewMotionPoll = null;
+
+    public motionId: number;
+
+    public isEditingPoll = false;
+
+    public contentForm: FormGroup;
+
+    public constructor(
+        title: Title,
+        protected translate: TranslateService,
```

Comment:
Remove the protected modifier. Please check all other components, if there is a protected modifier aswell.